docs/en/01_installation.md

```diff
@@ -44,18 +44,18 @@ In other cases, submissions could be expected to contain private data.
 
 If you want to use custom email templates set the following config option.
 
-```yaml
+```yml
 SilverStripe\UserForms\Model\UserDefinedForm:
   email_template_directory: mysite/templates/custom_userforms_emails/
 ```
 
-Any SilverStripe templates placed in your `email_template_directory` directory will be available for use with submission emails.
+Any Silverstripe templates placed in your `email_template_directory` directory will be available for use with submission emails.
```

Comment:
```suggestion
Any Silverstripe CMS templates placed in your `email_template_directory` directory will be available for use with submission emails.
```
Or just remove Silverstripe and say "any templates"

Reply:
Updated